src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEEventSymbol.cs

```diff
@@ -96,10 +96,13 @@ private enum Flags : byte
 
                 const int targetSymbolCustomModifierCount = 0;
                 var typeSymbol = DynamicTypeDecoder.TransformType(originalEventType, targetSymbolCustomModifierCount, handle, moduleSymbol);
-                var type = TypeSymbolWithAnnotations.Create(typeSymbol);
+
+                // We start without annotations
```

Comment:
Might be good to elaborate a bit as to _why_ we start without annotations

Reply:
Same for the other similar comments.

---
In reply to: [204937418](https://github.com/dotnet/roslyn/pull/28491#discussion_r204937418) [](ancestors = 204937418)